fast/stages/0-bootstrap/organization.tf

```diff
@@ -151,4 +156,40 @@ module "organization" {
       type                 = attrs.type
     }
   }
+  org_policies_data_path = var.factories_config.data_dir
+  org_policies = {
+    "iam.allowedPolicyMemberDomains" = {
+      rules = [
+        {
```

Comment:
`var.factories_config` can be null. Either check for null values or (better) make the variable non-nullable?

Reply:
added `nullable == false`, and users can provide an empty dir (or set `data_dir` to null the module would survive)